org.eclipse.wildwebdeveloper.tests/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Tests for WildWebDeveloper
 Bundle-SymbolicName: org.eclipse.wildwebdeveloper.tests
-Bundle-Version: 0.2.0.qualifier
+Bundle-Version: 0.2.1.qualifier
```

Comment:
Since it's a relatively noticeable addition, we'll bump to 0.3.0 to showcase there is more than maintenance fix.